app/models/concerns/user_concern.rb

```diff
@@ -226,6 +226,22 @@ def training_status(course)
       end
     end
 
+    def training_renewal_date(course)
+      return nil unless course.months_to_expiration.present?
+
+      login = Talentlms::Login.find_by(user: self, config: course.config)
+      return nil unless login
+
+      completion_date = Talentlms::CompletedTraining.find_by(course: course, login: login)&.completion_date
+      return nil unless completion_date
+
+      if Talentlms::Facade.expiration_duration_period == :days
+        completion_date + course.months_to_expiration
+      else
+        completion_date << course.months_to_expiration
```

Comment:
Does the shovel operator have some different behavior for dates?  Is this different from:
```
if Talentlms::Facade.expiration_duration_period == :days
         completion_date + course.months_to_expiration.days
       else
         completion_date + course.months_to_expiration.months
```

Reply:
The shovel operator adds months to a date, but I'm going to change it to your recommendation. For whatever reason this way of doing it wasn't coming to me when building this out.